bundle/manifests/metallboperator-metrics-reader_rbac.authorization.k8s.io_v1beta1_clusterrole.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
I think we can remove this file

Reply:
i think this one was also created automatically in the initial setup, should we remove it?
https://github.com/metallb/metallb-operator/blob/main/config/rbac/auth_proxy_client_clusterrole.yaml

Reply:
if you are not exposing metrics I think this is not needed (you can remove it from the kustomize and leave the file for future use)